Worlds Docs: Translations German (Clique, BK Sudoku, OoT)

ZeitHeld opened the pull request:
~~I still dunno, if that is how you do a PR, but I hope it's good enough~~

## What is this fixing or adding?
3 Games have been translated into german (and even tried to keep the formatting rules). The Setup/Guide as well as the game info page(?) (since it has `en_` in front of it)

**Additions:**
- BK-Sudoku
- Clique
- OoT (With a screenshot)

Only one Screenshot has been added to Ocarina of Time (could be used universally), regarding the Patch Download for ROMS.

BootsinSoots commented:
Thank you for your work! You did the PR right, good job. I advertised it a little so those who speak German can provide feedback (I do not speak it myself, so I'm not much help)

Jarno458 commented:
Well for BK sudoku with my limited German skiles, it seems alright, but you forgot something important for all 3 games
You need to add the new guides to the game its `WebWorld` often located in the `__init__.py`

black-sliver commented:
Can we finally request a doc review from berserker? :D

ZeitHeld commented:
> Well for BK sudoku with my limited German skiles, it seems alright, but you forgot something important for all 3 games You need to add the new guides to the game its `WebWorld` often located in the `__init__.py`

Ooh, I didn't really read that anywhere (tbh, going through the docs already is a bit obscure and stuff. Not quite obvious).
But I think I can look at it sooner or later if I can add it in by myself (shouldn't be too hard ig?)

ZeitHeld commented:
~~what did that-~~

uuhhh, anyways, I added the lines of code into the init's and also added some "tranlation flexibility" into some others, that weren't made for multiple setups

Though, I think I made the changes into my "main" branch on accident- lemme fix that

ZeitHeld commented:
Okay I think now it should be- (sorry about that)

Jarno458 commented:
Oke +1 for your enthusiasm, but you refactored a lot of init.py's, this is not idea as it put them in the scope of your PR and will require those maintainers to approve the PR in addition to the 3 maintainers of the guides you actually edited

Jarno458 commented:
While i admire your efforts to translate the game info page, i dont think there is currently any way to open those in a different language so that page will never be shown.. until a link to it is added somewhere

ZeitHeld commented:
Yeah, the edits were not initially ideal, but made as it is really not a big change- And kost of them are ones I try to translate too

And the info pages, yeaah- I noticed that later on too, but thought it could be done due to them as starting with "en"
If it eventually gets added (possibly along with translated settings pages) they could be used

black-sliver commented:
Please revert the changes to worlds that you didn't intend to modify. This will only create merge conflicts.

ZeitHeld commented:
> Please revert the changes to worlds that you didn't intend to modify. This will only create merge conflicts.

Okay, I think that should all done now
Sorry about the issue with that

black-sliver commented on worlds/clique/docs/guide_de.md:
```diff
@@ -0,0 +1,25 @@
+- Deine Post abholen.
+
+
+~~Solltest du auf irgendwelche Probleme in diesem Spiel stoßen, solltest du aufkeinenfall nicht **thephar** auf~~
```
auf keinen Fall
oder
keinenfalls

black-sliver commented on worlds/oot/docs/de_Ocarina of Time.md:
```diff
@@ -0,0 +1,41 @@
+
+## Wo ist die Seite für die Einstellungen?
+
+Die [Seite für die Spielereinstellungen dieses Spiels](../player-settings) enthält alle Optionen die man benötigt um
```
neuer link
```suggestion
Die [Seite für die Spielereinstellungen dieses Spiels](../player-options) enthält alle Optionen die man benötigt um
```

black-sliver commented on worlds/oot/docs/de_Ocarina of Time.md:
```diff
@@ -0,0 +1,41 @@
+
+## Was macht der Randomizer in diesem Spiel?
+
+Items welche der Spieler für gewöhnlich im Verlauf des Spiels erhalten würde, wurden umhergemischt. Die Logik bleit
```
```suggestion
Items, welche der Spieler für gewöhnlich im Verlauf des Spiels erhalten würde, wurden umhergemischt. Die Logik bleit
```

black-sliver commented on worlds/oot/docs/de_Ocarina of Time.md:
```diff
@@ -0,0 +1,41 @@
+## Welche Items und Bereiche werden gemischt?
+
+Alle ausrüstbare und sammelbare Gegenstände, sowie Munition können gemischt werden. Und alle Bereiche, die einen
+dieser Items enthalten könnten, haben (sehr wahrscheinlich) ihren Inhalt verändert. Goldene Skulltullas können ebenfalls
```
```suggestion
dieser Items enthalten könnten, haben (sehr wahrscheinlich) ihren Inhalt verändert. Goldene Skulltulas können ebenfalls
```

black-sliver commented on worlds/oot/docs/de_Ocarina of Time.md:
```diff
@@ -0,0 +1,41 @@
+
+## Welche Items können in sich in der Welt eines anderen Spielers befinden?
+
+Jedes dieser Items, die gemicht werden können, können in einer Multiworld auch in der Welt eines anderen Spieler's
```
```suggestion
Jedes dieser Items, die gemicht werden können, können in einer Multiworld auch in der Welt eines anderen Spielers
```

black-sliver commented on worlds/oot/docs/de_Ocarina of Time.md:
```diff
@@ -0,0 +1,41 @@
+
+## Wie sieht ein Item einer anderen Welt in OoT aus?
+
+Items die zu einer anderen Welt gehören, werden repräsentiert durch Zelda's Brief.
```
```suggestion
Items, die zu einer anderen Welt gehören, werden repräsentiert durch Zelda's Brief.
```

black-sliver commented on worlds/oot/docs/de_Ocarina of Time.md:
```diff
@@ -0,0 +1,41 @@
+
+## Einzigartige Lokale Befehle
+
+Die folgenden Befehle stehen nur im OoTClient, um mit Archipelago zu spielen, zur verfügung:
```
```suggestion
Die folgenden Befehle stehen nur im OoTClient, um mit Archipelago zu spielen, zur Verfügung:
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+- BizHawk: [BizHawk Veröffentlichungen von TASVideos](https://tasvideos.org/BizHawk/ReleaseHistory)
+  - Version 2.3.1 und später werden unterstützt. Version 2.9 ist empfohlen.
+  - Detailierte Installtionsanweisungen für BizHawk können über den obrigen Link gefunden werden.
+  - Windows-Benutzer müssen die Prerequisiten (Darunter `Visual Studio 2019`) installiert haben. Diese können ebenfalls über
```
```suggestion
  - Windows-Benutzer müssen die Prerequisiten installiert haben. Diese können ebenfalls über
```
Es sollte sich dabei nur um die Runtime handeln, nicht um das (volle) Visual Studio. Ist jedenfalls irrelevant: Entweder die prereqs sind schon zufällig installiert, oder sie müssen laut link installiert werden.

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+  - Windows-Benutzer müssen die Prerequisiten (Darunter `Visual Studio 2019`) installiert haben. Diese können ebenfalls über
+    den obrigen Link gefunden werden.
+- Der integrierte Archipelago-Client, welcher [hier](https://github.com/ArchipelagoMW/Archipelago/releases) installiert
+  werden kann. (wähle `Ocarina of Time Client` bei der Installation).
```
Checkbox wurde entfernt und wird immer automatisch installiert.
```suggestion
  werden kann.
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+## Konfigurieren von BizHawk
+
+Sobald Bizhawk einmal Installiert wurde, öffne **EmuHawk** und ändere die folgenen Einsteluungen:
```
```suggestion
Sobald Bizhawk einmal installiert wurde, öffne **EmuHawk** und ändere die folgenen Einsteluungen:
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+Sobald Bizhawk einmal Installiert wurde, öffne **EmuHawk** und ändere die folgenen Einsteluungen:
+
+- (≤ 2.8) Gehe zu `Config > Customize`. Wechlse zu dem `Advanced`-Reiter, wechsle dann den `Lua Core` von "NLua+KopiLua" zu
+  `"Lua+LuaInterface"`. Starte danach EmuHawk neu. Dies ist zwingend notwendig, damit die Lua-Skripts (mit denen man sich mit dem Client verbindet) ordnungsgemäß funktionieren.
```
```suggestion
  `"Lua+LuaInterface"`. Starte danach EmuHawk neu. Dies ist zwingend notwendig, damit die Lua-Scripts, mit denen man sich mit dem Client verbindet, ordnungsgemäß funktionieren.
```
Scripts oder Skripte

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+- Unter `Config > Customize > Advanced`, gehe sicher dass der Haken bei `AutoSaveRAM` ausgeählt ist, und klicke dann
+  den 5s-Knopf. Dies verringert die Wahrscheinlichkeit den Speicherfrotschritt zu verlieren, sollte der Emulator mal
+  abstürzen.
+- **(Optional)** Unter `Config > Customize`, kannst du die Haken in den "Run in background"
```
Glaub das Komma hier gehört nicht
```suggestion
- **(Optional)** Unter `Config > Customize` kannst du die Haken in den "Run in background"
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+  (Laufe weiter im Hintergrund) und "Accept background input" (akzeptiere Tastendruck im Hintergrund) Kästchen setzen.
+  Dies erlaubt dir das Spiel im Hintergrund weiter zu spielen, selbst wenn ein anderes Fenster aktiv ist. (Nützlich bei
+  mehreren oder eher großen Bildschrimen/Monitoren.)
+- Unter `Config > Hotkeys`, sind viele Hotkeys, die mit oft genuten Tasten belegt worden sind. Es wird empfohlen die
```
wie oben
```suggestion
- Unter `Config > Hotkeys` sind viele Hotkeys, die mit oft genuten Tasten belegt worden sind. Es wird empfohlen die
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+  Dies erlaubt dir das Spiel im Hintergrund weiter zu spielen, selbst wenn ein anderes Fenster aktiv ist. (Nützlich bei
+  mehreren oder eher großen Bildschrimen/Monitoren.)
+- Unter `Config > Hotkeys`, sind viele Hotkeys, die mit oft genuten Tasten belegt worden sind. Es wird empfohlen die
+  meisten (oder alle) Hotkeys zu deaktivieren. Dies kann schnell getan werden mit `Esc`.
```
```suggestion
  meisten (oder alle) Hotkeys zu deaktivieren. Dies kann schnell mit `Esc` erledigt werden.
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+  meisten (oder alle) Hotkeys zu deaktivieren. Dies kann schnell getan werden mit `Esc`.
+- Wird mit einem Kontroller gespielt, bei der Tastenbelegung (bei einem Laufendem Spiel, unter
+  `Config > Controllers...`), deaktiviere "P1 A Up", "P1 A Down", "P1 A Left", and "P1 A Right" und gehe stattdessen in
+  den Reiter `Analog Controls` um den Stick zu belegen, da sonst Probleme auftreten bei dem Zielen (mit dem Bogen oder
```
```suggestion
  den Reiter `Analog Controls` um den Stick zu belegen, da sonst Probleme beim Zielen auftreten (mit dem Bogen oder
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+  den Reiter `Analog Controls` um den Stick zu belegen, da sonst Probleme auftreten bei dem Zielen (mit dem Bogen oder
+  ähnliches). Y-Axis ist für Oben und Unten, und die X-Axis ist für Links und Rechts.
+- Unter `N64` setze einen Haken bei "Use Expansion Slot" (Benutze Erweiterungs-Slot). Dies wird benötigt damit
+  savestates/schnellspeichern funktioniert. (Das N64-Menü taucht nur auf **nach** dem laden einer N64-ROM.)
```
```suggestion
  savestates/schnellspeichern funktioniert. (Das N64-Menü taucht nur **nach** dem laden einer N64-ROM auf.)
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+### Was ist eine YAML-Datei und Warum brauch ich eine?
+
+Eine YAML-Datie enthält einen Satz an einstellbaren Optionen, welche Informationen den Zufallsgenerator liefert wie
```
Vielleicht einfacher so?
```suggestion
Eine YAML-Datie enthält einen Satz an einstellbaren Optionen, die dem Generator mitteilen, wie
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+erlaubt jeden Spieler eine personalisierte Erfahrung nach derem Geschmack. Damit kann auch jeder Spieler in einer
+Multiworld (des gleichen Spiels) völlig unterschiedliche Einstellungen haben.
+
+Für weitere Informationen besuche die Anleitung zum erstellen einer gewöhnlichen
+YAML-Datei: [Gewhönliche Multiworld Setup Antelitung](/tutorial/Archipelago/setup/de)
```
"Gewöhnlich" ist nicht gut übersetzt. "Allgemein" wäre richtig. /de-Link gibt es noch nicht, oder?
```suggestion
Für weitere Informationen, besuche die allgemeine Anleitung zum Erstellen einer
YAML-Datei: [Archipelago Setup Anleitung](/tutorial/Archipelago/setup/en)
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+### Woher bekomme ich eine YAML-Datei?
+
+Die Seite für die Spielereinstellungen auf dieser Website erlaubte es dir deine persönlichen Einstellungen nach
```
Tippfehler
```suggestion
Die Seite für die Spielereinstellungen auf dieser Website erlaubt es dir deine persönlichen Einstellungen nach
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+Die Seite für die Spielereinstellungen auf dieser Website erlaubte es dir deine persönlichen Einstellungen nach
+vorlieben zu konfigurieren und eine YAML-Datei zu exportieren.
+Seite für die Spielereinstellungen:
+[Seite für die Spielereinstellungen von Ocarina of Time](/games/Ocarina%20of%20Time/player-settings)
```
neuer link
```suggestion
[Seite für die Spielereinstellungen von Ocarina of Time](/games/Ocarina%20of%20Time/player-options)
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+### Überprüfen deiner YAML-Datei
+
+Wenn du deine YAML-Datei überprüfen möchtest, um sicher zu gehen dass sie funktioniert, kannst du dies auf der
```
```suggestion
Wenn du deine YAML-Datei überprüfen möchtest, um sicher zu gehen, dass sie funktioniert, kannst du dies auf der
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+Wenn du deine YAML-Datei überprüfen möchtest, um sicher zu gehen dass sie funktioniert, kannst du dies auf der
+YAML-Überprüfungsseite tun.
+YAML-Überprüfungsseite: [YAML-Überprüfungsseite](/mysterycheck)
```
neuer link
```suggestion
YAML-Überprüfungsseite: [YAML-Überprüfungsseite](/check)
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+(Der folgende Prozess ist bei den meisten ROM-basierenden Spielen sehr ähnlich.)
+
+Wenn du einer Multiworld beitrittst, wirst du gefordert eine YAML-Datei bei dem Host abzugeben. Ist dies getan, wird
+dich der Host der Multiworld mit (in der Regel) einem Link, der zu einem Raum führt. Dort sind alle Teilnehmenden
```
Zweiter Satz hier ist nicht vollständig?

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+
+### Verbinde zum Multiserver
+
+Sind einmal der Client und der Emulator gestartet, müssen sie nur noch mit einander Verbunden werden. Gehe dazu in
```
```suggestion
Sind einmal der Client und der Emulator gestartet, müssen sie nur noch miteinander verbunden werden. Gehe dazu in
```

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+fallen lassen) auf das EmuHawk-Fenster. (Alternativ kannst du die Lua-Konsole manuell öffnen, gehe dazu auf
+`Script > Open Script` und durchsuche die Ordner nach `data/lua/connector_oot.lua`.)
+
+Um den Client mit dem Multiserver zu verbinden, füge einfach `<Adresse>:<Port>` ein auf das Textfeld ganz oben im
+Client und drücke Enter oder "Connect" (verbinden). Wird Ein Passwort benötigt musst du es danach unten in das
```
```suggestion
Um den Client mit dem Multiserver zu verbinden, füge einfach `<Adresse>:<Port>` in das Textfeld ganz oben im
Client ein und drücke Enter oder "Connect" (verbinden). Wird ein Passwort benötigt, musst du es danach unten in das
```

black-sliver reviewed (commented):
I feel bad for the amount of suggestions.
German Komma be German Komma. Probably still missed some.

ZeitHeld commented:
> I feel bad for the amount of suggestions. German Komma be German Komma. Probably still missed some.

It's a diffciult langunge with it's forms of politeness, but I think/hope that after this weird sentence everything should be good.
Unless there is another issue found by someone or someone else
Don't worry about the amount. I shouldn't have done them at a late time lmao

black-sliver commented:
> but I think/hope that after this weird sentence everything should be good.

This is still to be done, right? (I see two conversations that are not marked as resolved yet.)

ZeitHeld commented on worlds/bk_sudoku/docs/de_Sudoku.md:
```diff
@@ -0,0 +1,21 @@
+## Wie werden Hinweise freigeschalten?
+
+Nach dem Lösen eines Sudokus wird das Spiel einen zufällig ausgewählten Hinweis freigeben für einen noch nicht
+gefundenen Check in dem Slot mit dem du verbunden bist.
```
ich habe das etwas kombiniert, und denke das klingt jetzt auch etwas besser

ZeitHeld commented:
Okay, I think that should be done now?
At least it seems to be resolved

black-sliver commented on worlds/bk_sudoku/__init__.py:
```diff
@@ -7,16 +7,25 @@
+
+    setup_en = Tutorial(
             tutorial_name='Setup Guide',
             description='A guide to playing BK Sudoku',
             language='English',
             file_name='setup_en.md',
             link='setup/en',
             authors=['Jarno']
-        )
-    ]
+    )
+    setup_de = Tutorial(
+            tutorial_name='Setup Anleitung',
+            description='Eine Anleitung um BK-Sudoku zu spielen',
+            language='Deutsch',
+            file_name='setup_de.md',
+            link='setup/de',
+            authors=['Held_der_Zeit']
+    )
```
Hm, this still has wrong indentation
```python
    setup_en = Tutorial(
        tutorial_name='Setup Guide',
        description='A guide to playing BK Sudoku',
        language='English',
        file_name='setup_en.md',
        link='setup/en',
        authors=['Jarno']
    )
    setup_de = Tutorial(
        tutorial_name='Setup Anleitung',
        description='Eine Anleitung um BK-Sudoku zu spielen',
        language='Deutsch',
        file_name='setup_de.md',
        link='setup/de',
        authors=['Held_der_Zeit']
    )
```
would be correct.

black-sliver commented on worlds/oot/docs/setup_de.md:
```diff
@@ -0,0 +1,108 @@
+(Der folgende Prozess ist bei den meisten ROM-basierenden Spielen sehr ähnlich.)
+
+Wenn du einer Multiworld beitrittst, wirst du gefordert eine YAML-Datei bei dem Host abzugeben. Ist dies getan,
+erhälst du (in der Regel) einem Link, von dem Host der Multiworld. Dieser führt dich zu einem Raum in dem alle
```
Glaub so:
```suggestion
erhälst du (in der Regel) einen Link vom Host der Multiworld. Dieser führt dich zu einem Raum, in dem alle
```

black-sliver reviewed (approved):
Looks good now. I DM'd phar and espeon to make sure they don't have any objections.
